flink-cdc-connect/flink-cdc-pipeline-connectors/flink-cdc-pipeline-connector-mysql/src/main/java/org/apache/flink/cdc/connectors/mysql/source/parser/CustomAlterTableParserListener.java

```diff
@@ -371,7 +374,10 @@ public void exitAlterByModifyColumn(MySqlParser.AlterByModifyColumnContext ctx)
                     Column column = columnDefinitionListener.getColumn();
                     Map<String, DataType> typeMapping = new HashMap<>();
                     typeMapping.put(column.name(), fromDbzColumn(column, tinyInt1isBit));
-                    changes.add(new AlterColumnTypeEvent(currentTable, typeMapping));
+                    AlterColumnTypeEvent alterColumnTypeEvent =
+                            new AlterColumnTypeEvent(currentTable, typeMapping);
+                    alterColumnTypeEvent.addColumnComment(column.name(), column.comment());
+                    changes.add(alterColumnTypeEvent);
```

Comment:
Please update the test of `MySqlPipelineITCase` to validate this change.

Reply:
ok